addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1436,8 +1439,8 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         const type = this._getFieldType();
 
         const list = document.createElement('we-list');
-        const optionText = select ? 'Option' : type === 'selection' ? 'Radio' : 'Checkbox';
-        list.setAttribute('string', `${optionText} List`);
+        const optionText = select ? _t("Option") : type === 'selection' ? _t("Radio") : _t("Checkbox");
+        list.setAttribute('string', _.str.sprintf(_t("%s List"), optionText));
```

Comment:
Note that from v17 onward this needs to be adapted to use the variadic _t instead of sprintf.

Reply:
And in any case: never use underscore anymore, it's gone in next versions. Prefer our sprintf utils here, although, here, it does not really matter as this line would have to be changed during forward-port anyway I guess :+1:

Reply:
I changed it to use our utils function here.